projects/documentation/.eleventy.js

```diff
@@ -60,43 +60,103 @@ export default function (eleventyConfig) {
         html: true,
     };
     const markdown = markdownIt(options).use(markdownItAnchors, {
-        level: [2, 3, 4],
         permalink: true,
         permalinkSymbol: '#',
-        permalinkAttrs: () => ({ 'aria-label': '§' }),
+        level: [2, 3, 4],
         renderPermalink: (slug, opts, state, idx) => {
-            const space = () =>
-                Object.assign(new state.Token('html_block', '', 0), {
-                    content: '&nbsp;',
-                });
+            // based on fifth version in
+            // https://amberwilson.co.uk/blog/are-your-anchor-links-accessible/
+            const linkContent = state.tokens[idx + 1].children[0].content;
+            const heading = state.tokens[idx];
+            const tag = heading.tag;
+
+            // Create the openning <div> for the wrapper
+            const headingWrapperTokenOpen = Object.assign(
+                new state.Token('div_open', 'div', 1),
+                {
+                    attrs: [['class', `headerContainer`]],
+                }
+            );
+            // Create the closing </div> for the wrapper
+            const headingWrapperTokenClose = Object.assign(
+                new state.Token('div_close', 'div', -1),
+                {
+                    attrs: [['class', `headerContainer`]],
+                }
+            );
+            const size =
+                tag === 'h2'
+                    ? 'XL'
+                    : tag === 'h3'
+                      ? 'L'
+                      : tag === 'h4'
+                        ? 'M'
+                        : tag === 'h5'
+                          ? 'S'
+                          : tag === 'h6'
+                            ? 'XS'
+                            : '';
+            const classes =
+                size === ''
+                    ? ''
+                    : `spectrum-Heading spectrum-Heading--size${size}`;
 
-            const linkTokens = [
-                Object.assign(new state.Token('link_open', 'sp-link', 1), {
+            heading.attrs = [
+                ...heading.attrs,
+                ['class', `header-heading ${classes}`],
+            ];
+
+            // Create the tokens for the full accessible anchor link
```

Comment:
As a follow-up from our discussions, could we double-check the best way to display these anchor links? For example, should we consider having them always visible, or perhaps toggle visibility for non-hover inputs?

Reply:
@jnurthen I like this idea. What are your thoughts on making them always visible?